docs/docs/sample-scripts/spo/list-all-list-folders-itemcount/index.md

```diff
@@ -30,7 +30,7 @@ List all Lists, the folders and sub folders in a given site, and output the item
       $folders = m365 spo folder list -u $webUrl --parentFolderUrl $folderUrl -o json | ConvertFrom-Json
 
       foreach ($folder in $folders) {
-        $folderStats = m365 spo folder get -u $webUrl --folderUrl $folder.ServerRelativeUrl -o json | ConvertFrom-Json
+        $folderStats = m365 spo folder get -u $webUrl --url $folder.ServerRelativeUrl -o json | ConvertFrom-Json
```

Comment:
I know it will work but it just seems kinda weird to mix short with long option naming 😅. Could we align 😉?

```suggestion
        $folderStats = m365 spo folder get -u $webUrl -f $folder.ServerRelativeUrl -o json | ConvertFrom-Json
```